lua/core/mappings.lua

```diff
@@ -193,6 +193,58 @@ if is_available "toggleterm.nvim" then
   maps.t["<C-'>"] = maps.n["<F7>"]
 end
 
+if is_available "nvim-dap" then
+  -- modified function keys found with `showkey -a` in the terminal to get key code
+  -- run `nvim -V3log +quit` and search through the "Terminal info" in the `log` file for the correct keyname
+  maps.n["<F5>"] = {
+    function()
+      if is_available "nvim-dap-ui" then require("dapui").open() end
+      require("dap").continue()
+    end,
+    desc = "Debugger: Start",
+  }
+  maps.n["<F17>"] = {
+    function()
+      if is_available "nvim-dap-ui" then require("dapui").close() end
+      require("dap").terminate()
+    end,
+    desc = "Debugger: Stop",
+  }
+  maps.n["<F29>"] = { function() require("dap").restart_frame() end, desc = "Debugger: Restart" }
+  maps.n["<F6>"] = { function() require("dap").pause() end, desc = "Debugger: Pause" }
+  maps.n["<F9>"] = { function() require("dap").toggle_breakpoint() end, desc = "Debugger: Toggle Breakpoint" }
+  maps.n["<F10>"] = { function() require("dap").step_over() end, desc = "Debugger: Step Over" }
+  maps.n["<F11>"] = { function() require("dap").step_into() end, desc = "Debugger: Step Into" }
+  maps.n["<F23>"] = { function() require("dap").step_out() end, desc = "Debugger: Step Out" }
+  maps.n["<leader>Db"] = { function() require("dap").toggle_breakpoint() end, desc = "Toggle Breakpoint (F9)" }
+  maps.n["<leader>DB"] = { function() require("dap").clear_breakpoint() end, desc = "Clear Breakpoints" }
```

Comment:
```suggestion
  maps.n["<leader>DB"] = { function() require("dap").clear_breakpoints() end, desc = "Clear Breakpoints" }
```

On my end only `clear_breakpoints` exist